kalman_walk_paper.tex

```diff
@@ -268,19 +268,19 @@ \section*{Introduction}
 % predicts many general patterns from the speciation literature \citep{fraisse2016genetics}.
 It has also been shown that population structure
 can allow long-term stable coexistence of incompatible genotypes encoding identical phenotypes \citep{phillips1996maintenance}.
-However, previous simulations of system drift in regulatory sequences \citep{tulchinsky2014hybrid}
-and a regulatory cascade \citet{porter2002speciation}
+Despite the findings of these quantitative genetics models, previous simulations of system drift in regulatory sequences \citep{tulchinsky2014hybrid}
+and a regulatory cascade \citep{porter2002speciation}
 found that speciation could be rapid under directional selection
 but only equivocal support for speciation under models of purely neutral drift.
 %Additionally, \citet{fierst} observed speciation due to an increase of segregation variance under balancing selection, in models of weak to moderate, but not strong epistasis.
+It is known that hybrid incompatibility can result from the increased segregation variance of a quantitative trait \citep{slatkin1994segregation, chevin2014niche}. Here, by focusing on the explicit biological function or physiology of a phenotype, we show that the evolution of even extremely simple phenotypes can appropriately be approximated by quantitative genetics. 
+This is because, like the genetic code, the genotype-phenotype ``code'' is fundamentally degenerate -- as we make explicit -- and thus ammendable to quantitative genetic analysis. 
```

Comment:
This statement is a tad too general - it will need at least a caveat saying "it seems likely that" or some such.